portfolio/src/main/java/com/google/sps/data/Grid.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public class Grid {
+  private final int row;
+  private final int col;
+
+  public Grid(int row, int col){
+      this.row = row;
+      this.col = col;
+  }
+
+  public double getRow(){
```

Comment:
Instead of these get methods, you could just make the row/col properties public.

Reply:
Ok, I changed it, thank you, Amin!